Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -103,5 +97,167 @@ public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util)
             await _dbContext.SaveChangesAsync();
         }
         #endregion
+
+        #region Category selection
+
+        public async Task AddTestCategoriesAsync(int testId, List<CategoryAC> categoryAcList)
+        {
+            var testCategoryList = new List<TestCategory>();
+            var testCategories = await _dbContext.TestCategory.ToListAsync();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'testCategories' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)